src/util/deprecated.js

```diff
@@ -0,0 +1,5 @@
+export default function deprecated (elem, oldUsage, newUsage) {
+  // TODO: add logic to only warn in dev mode
```

Comment:
A common pattern seems to be `process.env.NODE_ENV`. Most libs tend to favour this and it gets built out of the dist files.